src/main/java/org/apache/commons/compress/compressors/zstandard/ZstdUtils.java

```diff
@@ -134,4 +157,21 @@ public static void setCacheZstdAvailablity(final boolean doCache) {
     /** Private constructor to prevent instantiation of this utility class. */
     private ZstdUtils() {
     }
+
+    /**
+     * Reads uncompressed data stream and writes it compressed to the output
+     *
+     * @param input the data stream with uncompressed data
+     * @param output the data stream for compressed output
+     * @return the compressed size
+     * @throws IOException throws the exception which could be got from from IOUtils.copyLarge()
+     *         or ZstdCompressorOutputStream constructor
+     */
+    public static long readAndCompressWrite(InputStream input, OutputStream output) throws IOException {
+        final InnerNotClosingOutputStream outStream = new InnerNotClosingOutputStream(output);
+        final ZstdCompressorOutputStream outputStream = new ZstdCompressorOutputStream(outStream, 3, true);
```

Comment:
The value `3` looks like a magic number. A comment should state the reason to use 3 instead of another number.